assignments/function-conversion.js

```diff
@@ -5,26 +5,50 @@
 // };
 // myFunction();
 
+const myFunction = () => {console.log("Function was invoked!")}
+
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+const anotherFunction = (param) => param
+
+console.log(anotherFunction("boi"))
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+const add = (param1, param2) => param1 + param2
+
+console.log(add(2,8))
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+const subtract = (param1, param2) => param1 - param2
+
+console.log(subtract(5,6))
 
 // Stretch
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+let myArray = [1,3,5,7,100];
+
+const triple = (regulArray) => {
```

Comment:
another way to write this would be to do the following:
const triple = exampleArray.map((num) => num * 3);